apps/right-to-rent-check/behaviours/caseworker-email.js

```diff
@@ -5,8 +5,12 @@ const path = require('path');
 
 const getSubjectWithTotalTenants = (model, translate) => {
   const totalTenants = model.tenants.length;
```

Comment:
I know this isn't your code, but this should be using a getter `model.get('tenants').length`

Reply:
actually it was me originally, why does it make a difference?

Reply:
I tried `model.get('tenants').length` but I get `model.get is not a function`, is this model the same as a hof-model?